bin/lolcommits

```diff
@@ -116,7 +87,9 @@ def do_capture
     :config          => configuration
   }
 
-  fork_me? do
```

Comment:
This is a good idea, and fits in with the overall structure.  I think it should be done as part of a `Configuration` refactor (as a new PR), so let's start some notes for that.